linux_os/guide/system/software/disk_partitioning/partition_for_tmp/rule.yml

```diff
@@ -21,9 +21,7 @@ identifiers:
 references:
     stigid@ol7: OL07-00-021340
     anssi: BP28(R12)
-    cis@rhel8: 1.1.2
-    cis@ubuntu1804: 1.1.2
-    cis@ubuntu2004: 1.1.2
+    cis: 1.1.2
```

Comment:
Tempting as it may be to unify these references, I lean towards not merging them.
There is no guarantee that CIS profiles added in the future will match these references numbers.

Reply:
Wouldn't it make sense to unify the references and then just have the outliers be overrides? This is done is many, many places with references defaulting to RHEL and then @ubuntu-whatever being overrides.

Reply:
Yes, it does make sense to unify references until there are deviations, and they should always be unified unless it deviates. Deviations are exceptions not the rule.

Reply:
I mainly would like to avoid inconsistencies.
If the reference is not product specific it will show up in every product, even for products that don't have a CIS profile implemented.
Another point is that when adding a new CIS profile to a product, one will have to process every rule with a `cis` reference in it, either to accept the reference or remove it.

Reply:
> This is done is many, many places with references defaulting to RHEL and then @ubuntu-whatever being overrides.

Well, yes, maybe it would not be a problem to unify them.

> Deviations are exceptions not the rule.

When it comes to sharing descriptions, checks and fixes, it makes sense. A lot of code and practices is shared among distros.
I know that CIS benchmarks have some kind of hierarchy and relationship between them, So unifying them may not be a big problem, and my concerns are not relevant.

Reply:
I'm fine with either, but the draft is a living specification and it often depends on what version of a particular OS spec you're looking at (e.g., v1.1.0 of the Ubuntu 20.04 spec changes a number of section numbers than v1.0.0). 

My concern is someone going ahead and changing the `cis` reference because, e.g., RHEL changed, but e.g., Ubuntu didn't, without looking at all products the rule currently applies to.

If we make deviations the exception, not the rule, we need to make sure updating the default involves more cross-checking. :-)

My 2c.

Reply:
@cipherboy So you are slightly in favor of defaulting to specific `cis` references then?

Reply:
Yeah, on the other side as an intern back in '18 I definitely would've preferred a single `cis` reference with OS-specific overrides when necessary.

But now having seen more of the CIS process, I don't think that is correct, as helpful as it would be ;-) 

Each OS needs to go through CIS process and have a spec published before a CIS reference becomes valid for it. So having no default `cis` reference makes it explicit and obvious that this has been done for a given OS/rule combination. 

My 2c.